neutrons/SNAPRed
# squashed changes: JSONForm refactor, Reduction Workflow Orchestration…

…, Continue Anyway flow changes

## Description of work

This pr contains:
- some refactors to various ui elements to remove the JSONWidget stuff we dont really use anymore
- Implementation of a basic reduction workflow
- extension of the continueAnyway ui functionality to react to certain continue anyway scenarios

## To test

1. Prep a modern calibration and normalization, if this is troublesome ping me to lend you a copy through my shared folder.
2. Open SNAPRed pointing at your local files
3. Travel to the Reduction page
4. Submit runnumber `46680`
5. Confirm that reduction runs and completes.

Not completely relevant to the orchestration but if you like to verify data: 
Things to look out for when you setup your calibration:

![image](https://github.com/neutrons/SNAPRed/assets/68125095/475512bf-d624-40f3-a129-97758be1c24a)
you'll want to edit paritcleBounds in the record and parameters json to be about this number

You will also want to ensure your vanadium isnt negative or something else funny or you will get a flipped graph 
the target output should look something like this:
![image](https://github.com/neutrons/SNAPRed/assets/68125095/b150f643-d96f-490c-9bde-3244c21cdbe4)


## Link to EWM item


[EWM#4880](https://ornlrse.clm.ibmcloud.com/ccm/web/projects/Neutron%20Data%20Project%20(Change%20Management)#action=com.ibm.team.workitem.viewWorkItem&id=4880)
